ethereum/networks/src/test/java/tech/pegasys/teku/networks/Eth2NetworkConfigurationTest.java

```diff
@@ -104,6 +107,36 @@ public void applyNamedNetworkDefaults_shouldOverwritePreviouslySetValues() {
     }
   }
 
+  @Test
+  public void bootnodesFromMainnetNetworkDefaults_CanBeParsed() {
+    final Eth2NetworkConfiguration config =
+        Eth2NetworkConfiguration.builder().applyMainnetNetworkDefaults().build();
```

Comment:
We can pick up all the networks with a single parameterised test like `build_shouldBuildKnownNetworks` does at the top of the file.  

It comes out something like:

```
  @ParameterizedTest
  @MethodSource("getDefinedNetworks")
  public void bootnodesFromPraterNetworkDefaults_CanBeParsed(final Eth2Network network, final NetworkDefinition networkDefinition) {
    final Eth2NetworkConfiguration config =
        Eth2NetworkConfiguration.builder(network).build();
    final Eth2NetworkConfiguration.Builder networkConfigBuilder =
        Eth2NetworkConfiguration.builder();
    networkDefinition.configure(networkConfigBuilder);
```

Reply:
I applied this suggestion, and indeed tests are simpler and more expressive now. Thank you!